packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -104,10 +115,11 @@ export const getProgressColor: GaugePropsGetColor = ({
  */
 
 export function Gauge(props: GaugeProps) {
+  const [hoverRef, isHovering] = useHover() as any;
```

Comment:
Can we remove this `any`?

Reply:
@benjdlambert Hey Ben! I removed the `any`. When you have a chance. Can you please review? Thanks